code/game/objects/structures/watercloset.dm

```diff
@@ -47,12 +51,29 @@
 	if(I.is_crowbar())
 		to_chat(user, "<span class='notice'>You start to [cistern ? "replace the lid on the cistern" : "lift the lid off the cistern"].</span>")
 		playsound(loc, 'sound/effects/stonedoor_openclose.ogg', 50, 1)
-		if(do_after(user, 30))
+		if(do_after(user, 15))
 			user.visible_message("<span class='notice'>[user] [cistern ? "replaces the lid on the cistern" : "lifts the lid off the cistern"]!</span>", "<span class='notice'>You [cistern ? "replace the lid on the cistern" : "lift the lid off the cistern"]!</span>", "You hear grinding porcelain.")
 			cistern = !cistern
 			update_icon()
 			return
 
+	if(I.is_wrench())
+		water_supply = !water_supply
+		to_chat(user, "You [water_supply ? "close" : "open"] \the [src] water supply. It won't refill after being flushed now.")
+		return //So you don't put the wrench insides
+
+	if(istype(I, /obj/item/weapon/reagent_containers))
+		if(cistern)
+			if(I.reagents.total_volume < 1)
+				user << "\The [I] is empty."
```

Comment:
to_chat(user, instead of <<

Reply:
what are you doing reviewing pr's 'round here, partner